pkg/lib/cockpit-components-terminal.jsx

```diff
@@ -130,6 +138,37 @@ export class Terminal extends React.Component {
         window.removeEventListener('resize', this.onWindowResize);
     }
 
+    setText() {
+        try {
+            navigator.clipboard.readText()
+                    .then((text) => {
+                        this.props.channel.send(text);
+                    })
```

Comment:
Please let's avoid the unnecessary `{ ; }` if the arrow function body is a single expression (like in all three cases here).